ui/src/copainput.tsx

```diff
@@ -97,50 +110,48 @@ export function CopaInput(props: any) {
     props.setUseContainerdChecked(event.target.checked);
   };
 
+  const handleSelectedImageChange = (event: any, newValue: string | null) => {
+    props.setSelectedImage(newValue);
+    if (newValue !== null) {
```

Comment:
can we add the same error checking we had before for the input. For example, we shouldn't be able to scan with no input.

Reply:
this has been fixed by running the validateInput function and checking the result whenever the scan image button is clicked